tests/test_oauth2client.py

```diff
@@ -989,7 +989,7 @@ def test_construct_authorize_url(self):
     self.assertEqual('code', q['response_type'][0])
     self.assertEqual('foo', q['scope'][0])
     self.assertEqual(OOB_CALLBACK_URN, q['redirect_uri'][0])
-    self.assertEqual('offline', q['access_type'][0])
```

Comment:
Why remove this check? You can still check both the `access_type` and the `state`.